magento/inventory
# MSI-874: Api-Functional Tests For MSI are Failed.

### Description
Fix failed MSI REST and SOAP tests.

### Fixed Issues (if relevant)
1. magento-engcom/msi#874: Api-Functional Tests For MSI are Failed.

### Manual testing scenarios
1. Install Magento with MSI.
2. Configure dev/test/api-functional/phpunit_rest.xml and dev/test/api-functional/phpunit_soap.xml
3. Run api-function with REST adapter ex:
```
#!/bin/bash

RED='\e[1;31m'
GREEN='\e[1;32m'
BLUE='\e[1;34m'
ORANGE='\e[1;33m'
NC='\e[0m' # No Color

echo -e "${BLUE}Running MSI testsuite...${NC}";
php $PWD/vendor/phpunit/phpunit/phpunit --configuration $PWD/dev/tests/api-functional/phpunit_rest.xml --testsuite "msi"
```
4. Check all test are green. No failures or errors occured.
5. Run api-functional tests with SOAP adapter. ex:
```
#!/bin/bash

RED='\e[1;31m'
GREEN='\e[1;32m'
BLUE='\e[1;34m'
ORANGE='\e[1;33m'
NC='\e[0m' # No Color

echo -e "${BLUE}Running MSI testsuite...${NC}";
php $PWD/vendor/phpunit/phpunit/phpunit --configuration $PWD/dev/tests/api-functional/phpunit_soap.xml --testsuite "msi"
```
6. Check no errors and failures occured. Some tests may be skipped.
